config/manager/controller_manager_config.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
+kind: ControllerManagerConfig
+health:
+  healthProbeBindAddress: :8081
+metrics:
+  bindAddress: 127.0.0.1:8080
+
+leaderElection:
+  leaderElect: true
+  resourceName: 811c9dc5.crowdstrike.com
```

Comment:
Is this a static name? Does it mean something?

Reply:
Yes, this is a static name for the kube controller. Most likely used by the kube-scheduler. It is the name of resource object that is used for locking during leader election and is part of the scaffolding provided by the operator-sdk.